std/bitmanip.d

```diff
@@ -56,14 +56,22 @@ import std.range.primitives;
 public import std.system : Endian;
 import std.traits;
 
-private string myToString(ulong n)
+private string myToString(ulong n) pure @safe
 {
     import core.internal.string : UnsignedStringBuf, unsignedToTempString;
     UnsignedStringBuf buf;
     auto s = unsignedToTempString(n, buf);
-    return cast(string) s ~ (n > uint.max ? "UL" : "U");
+    return s ~ (n > uint.max ? "UL" : "U");
```

Comment:
I think with the comment "use pure to implicitly cast to string" this was much better

Reply:
Fixed.